lib/test/t_CovarianceModel_std.cxx

```diff
@@ -179,6 +179,22 @@ int main(int, char *[])
       test_model(myModel);
     }
 
+    {
+      Scalar scale = 3.5;
+      Scalar amplitude = 1.5;
+      Point scalePoint(1);
+      scalePoint[0] = scale;
+      Point amplitudePoint(1);
+      amplitudePoint[0] = amplitude;
+      SquaredExponential myOneDimensionalKernel(scalePoint, amplitudePoint);
+      IsotropicCovarianceModel myIsotropicKernel(myOneDimensionalKernel, 2);
+      assert_almost_equal(myIsotropicKernel.getAmplitude()[0], amplitude, 0.0, 1e-6);
```

Comment:
Test that the IsotropicCovarianceModel(s,t) is consistent with values obtained with anisotropic geometric CovarianceModels with all scale parameters equal.